src/OpenDiffix.Core/QueryValidator.fs

```diff
@@ -30,21 +30,34 @@ let private validateAllowedAggregates query =
   |> visitAggregates (
     function
     | FunctionExpr (AggregateFunction (Count, _), _) -> ()
+    | FunctionExpr (AggregateFunction (CountNoise, _), _) -> ()
     | FunctionExpr (AggregateFunction (Sum, _), _) -> ()
     | FunctionExpr (AggregateFunction (_otherAggregate, _), _) ->
-      failwith "Only count and sum aggregates are supported in anonymizing queries."
+      failwith "Only count, count_noise and sum aggregates are supported in anonymizing queries."
     | _ -> ()
   )
 
 let private allowedCountUsage query =
+  let message =
+    "Only count(*), count(column), count_noise(*), count_noise(column)"
+    + " and count(distinct column) are supported in anonymizing queries."
```

Comment:
This type of error message is not scalable: the more aggregators we support, the longer it gets. And we also need to update it each time. Something more generic would be better here, like: "Invoked aggregator is not supported in anonymizing queries.".

Reply:
yeah, was thinking the same and wondering is it already the time to cut this. Current form has the advantage of spelling out what is supported, but that won't last.